wfe/web-front-end.go

```diff
@@ -624,13 +625,30 @@ func (wfe *WebFrontEndImpl) RevokeCertificate(response http.ResponseWriter, requ
 	}
 }
 
+func (wfe *WebFrontEndImpl) logCsr(remoteAddr string, cr core.CertificateRequest, registration core.Registration) {
+	var csrLog = struct {
+		RemoteAddr   string
+		CsrBase64    string
+		Registration core.Registration
+	}{
+		RemoteAddr:   remoteAddr,
+		CsrBase64:    base64.StdEncoding.EncodeToString(cr.Bytes),
```

Comment:
(Look for "base64" in the [docs on Marshal](https://golang.org//pkg/encoding/json/#Marshal))